routers/private/manager.go

```diff
@@ -16,6 +17,7 @@ import (
 	"code.gitea.io/gitea/modules/queue"
 	"code.gitea.io/gitea/modules/setting"
 	"code.gitea.io/gitea/modules/web"
+	"xorm.io/xorm"
```

Comment:
IMO it's better to avoid making router handler depend on `xorm` directly. And the SetLogSQL should take affect on all database connections (if there are not only one in the future), instead of the current context (which is unclear). Maybe make the router handler call a `db.SetLogSQL()`?